tuf/download.py

```diff
@@ -257,14 +231,34 @@ def _download_file(url, required_length, STRICT_REQUIRED_LENGTH=True):
   temp_file = securesystemslib.util.TempFile()
 
   try:
-    # Open the connection to the remote file.  _open_connection() can raise
-    # socket connection exceptions (such as SSLError).  Connection errors of
-    # this kind can be minimized by adjusting the socket timeout in
-    # settings.py.
-    connection = _open_connection(url)
+    # Attach some default headers to every Session.
+    _session.headers.update({
+        # Tell the server not to compress or modify anything.
+        # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding#Directives
+        'Accept-Encoding': 'identity',
+        # The TUF user agent.
+        # TODO: Attach version number, which cannot be easily found right now.
```

Comment:
We definitely need something defined here, and add it to part of our release process...